lib/std-gen-seed/std-gen-seed.cabal

```diff
@@ -0,0 +1,64 @@
+cabal-version:   3.0
+name:            std-gen-seed
+version:         0.1.0.0
+synopsis:        modules to support applications for the cardano wallet
+license:         Apache-2.0
+author:          HAL, Cardano Foundation
+maintainer:      hal@cardanofoundation.org
+build-type:      Simple
+extra-doc-files: CHANGELOG.md
+
+library
+  default-language:   Haskell2010
+  default-extensions:
+    NoImplicitPrelude
+    DerivingStrategies
+    OverloadedStrings
+
+  ghc-options:
+    -O2 -Wall -Wincomplete-record-updates -Wincomplete-uni-patterns
+    -Wunused-foralls -Wunused-foralls -fprint-explicit-foralls
+    -fprint-explicit-kinds -Wcompat -Widentities
+    -Werror=incomplete-patterns -Wredundant-constraints
+    -Wpartial-fields -Wtabs -fhelpful-errors -fprint-expanded-synonyms
+    -fwarn-unused-do-bind -fwarn-incomplete-uni-patterns
+    -freverse-errors
+
+  exposed-modules:
+    System.Random.StdGenSeed
+
+  hs-source-dirs:     src
+  build-depends:
+    , base
+    , random
+    , MonadRandom
+    , OddWord
+    , quiet
+    , splitmix
+
+test-suite unit
+  default-language:
+      Haskell2010
+  default-extensions:
+      NoImplicitPrelude
+      OverloadedStrings
+  ghc-options:
+      -threaded -rtsopts
+      -Wall -O2 -Werror
```

Comment:
Similar to earlier comment, we should probably not specify `-O2` here unless it's for a release build.